notebooks/intercomparison-covermaps.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p>Is this meant to be filled out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>